roles/keycloak_quarkus/templates/keycloak-sysconfig.j2

```diff
@@ -1,6 +1,10 @@
 {{ ansible_managed | comment }}
+{% if not keycloak_quarkus_purge_admin_credentials_after_bootstrapping or not keycloak_quarkus_internal_bootstrapped %}
 KEYCLOAK_ADMIN={{ keycloak_quarkus_admin_user }}
 KEYCLOAK_ADMIN_PASSWORD='{{ keycloak_quarkus_admin_pass }}'
+{% else %}
+{{ keycloak.bootstrap_mnemonic }}
```

Comment:
Let's always add this line no matter what; it can turn useful in other scenarios.